tests/torch/nas/models/synthetic.py

```diff
@@ -335,6 +335,9 @@ class TwoSequentialFcLNTestModel(nn.Module):
     # fc1 -> ln1 -> fc2 -> ln2
     #
     INPUT_SIZE = [1, 1]
+    IMPORTANCE = {
+        "fc1.weight": torch.Tensor([[0.9], [0.1]]),
```

Comment:
Is there need for torch.Tensor? can it be expressed by list?

Reply:
Yes, it should be a tensor because it has the same structure/shape as the original model but it contains weight importance for reordering.